Formula/slirp4netns.rb

```diff
@@ -40,9 +40,8 @@ def install
     resource("test-api-socket").stage (testpath/"test")
     # The test secript requires network namespace to run, which is not available on Homebrew CI.
     # So here we check the error messages.
-    # Specifically, the test script will try for 40 times and fail with 1.
-    # So we match the output corresponding to the last attempt.
+    # Actually it does not print anything so we just check for the exit value.
     output = shell_output("bash ./test/test-slirp4netns-api-socket.sh", 1)
```

Comment:
```suggestion
    output = shell_output("bash ./test/test-slirp4netns-api-socket.sh 2>&1", 1)
```
The output might be going to stderr.

Reply:
Thank you so much